test/resources.test.ts

```diff
@@ -0,0 +1,17 @@
+import assert = require("assert");
+import { IconName, Resources } from "../src/extension/resources";
+import * as fs from "fs/promises";
+import * as vscode from "vscode";
+
+describe("The resources", () => {
+  const resources = new Resources(
+    vscode.extensions.getExtension("BazelBuild.vscode-bazel").extensionPath,
+  );
+  it("provides valid icon paths", async () => {
```

Comment:
for this test case to provide the intended test coverage, wouldn't we need to run the tests against the bundled code? Which would bring us back to https://github.com/bazel-contrib/vscode-bazel/pull/421#discussion_r1807583031

Reply:
At first I was only going to prevent the typo or future inconsistency when we add/remove/adjust the icons.

I still not get how the extension would be installed in `.vscode-test.js`, however when I print out the `extensionPath` it points to the project root. I think we should adjust the test config in #421 to make it test against the `dist/` bundle.